### PR TITLE
refactor: update all lodash imports

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

Update all usages of `import * as _ from 'lodash'` to explicitly import what we are actually using from lodash. This should impact the bundle size in a positive way as the former style ends up including all of `lodash` code, while the latter helps webpack with the tree shaking. I do not have numbers for this though.

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [ ] Added/updated relevant unit test(s) (and ran `yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/reports/assessment-report-model-builder.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { Assessment } from 'assessments/types/iassessment';
 import { ManualTestStatus } from 'common/types/manual-test-status';
 import { AssessmentData, AssessmentStoreData, TestStepInstance } from 'common/types/store-data/assessment-result-data';
 import { TabStoreData } from 'common/types/store-data/tab-store-data';
-import * as _ from 'lodash';
+import { keys } from 'lodash';
+import { find } from 'office-ui-fabric-react';
```
Comment (lisli1) [inline src/reports/assessment-report-model-builder.ts]: I'm curious why the change here uses `find` from `office-ui-fabric-react` instead of `lodash`?

Comment (smoralesd) [reply src/reports/assessment-report-model-builder.ts]: Good catch! Thank for finding this.
